DependencyInjection/SuluFormExtension.php

```diff
@@ -207,5 +211,25 @@ public function load(array $configs, ContainerBuilder $container)
             $container->setAlias(FormTokenController::class, 'sulu_form.form_token_controller')
                 ->setPublic(true);
         }
+
+        $this->configureHelper($config, $container);
+    }
+
+
+    private function configureHelper(array $config, ContainerBuilder $container)
+    {
+        $helper = $config['mail']['helper'];
+        if (\method_exists($container, 'resolveEnvPlaceholders')) {
+            $helper = $container->resolveEnvPlaceholders($helper, true);
+        }
+        $container->setParameter('sulu.mail.helper', $helper);
+
+        if ($helper == Configuration::MAILER_HELPER) {
+            $container->setAlias('sulu.mail.helper', 'sulu.mail.mailer');
+        } elseif ($helper == Configuration::NULL_HELPER) {
+            $container->setAlias('sulu.mail.helper', 'sulu_mail.null_helper');
+        } elseif ($helper == Configuration::SWIFT_MAILER_HELPER) {
+            $container->setAlias('sulu.mail.helper', 'sulu.mail.swift_mailer');
+        }
```

Comment:
This can be simplified to:

```suggestion
        $container->setAlias('sulu.mail.helper', 'sulu.mail.' . $helper);
```

This make it also extendable for the third party provide there own mailer.